Example/Glia Sample Integration/Presenters/ContentViewPresenter.swift

```diff
@@ -98,13 +132,47 @@ class ContentViewPresenter: ObservableObject {
             fontStyle: .default,
             showsPoweredBy: true
         )
-        theme.chat.backgroundColor = .white
-        theme.minimizedBubble.badge?.backgroundColor = .systemPink
+        theme.chat.backgroundColor = .fill(color: .white)
+        theme.minimizedBubble.badge?.backgroundColor = .fill(color: .systemPink)
         theme.minimizedBubble.badge?.fontColor = .white
         theme.minimizedBubble.badge?.font = .systemFont(ofSize: 14)
 
         return theme
     }
+    
+    func checkQueueEngagementsAvailability() {
+        Glia.sharedInstance.listQueues { [weak self] result in
+            switch result {
+            case .success(let queues):
+                let queue = queues.first(where: { $0.id == self?.gliaConfiguration.queueId })
+                self?.queue = queue
+                self?.isVideoAvailable = queue?.isMediaTypeSupported(.video) ?? false
+                self?.isAudioAvailable = queue?.isMediaTypeSupported(.audio) ?? false
+                self?.isChatAvailable = queue?.isMediaTypeSupported(.text) ?? false
+                self?.isAsyncMessagesAvailable = queue?.isMediaTypeSupported(.messaging) ?? false
+            case .failure(let error):
+                print("Error has happened: \(error.localizedDescription)")
+            }
+        }
+    }
+    
+    // Configure listening of Glia's events
+    func eventListener() {
+        Glia.sharedInstance.onEvent = { event in
+            switch event {
```

Comment:
I think adding comment here will be helpful - something like "This is an example of how events are handled in Glia Widgets", otherwise this code does not seem to do much.